learning_resources_search/api.py

```diff
@@ -494,10 +494,12 @@ def percolate_matches_for_document(document_id):
         Percolate(field="query", index=index, id=str(document_id))
     ).execute()
     percolate_ids = [result.id for result in results.hits]
+
     if len(percolate_ids) > 0:
-        learning_resource_percolated.send_robust(
-            sender=LearningResource,
-            instance=resource,
+        pm = get_plugin_manager()
+        hook = pm.hook
+        hook.document_percolated(
```

Comment:
Can you move this to a new `learning_resources_search.utils` function?

```python
def document_percolated_actions(resource, percolated_queries):
    """
    Trigger plugins when a LearningResource search document is percolated
    """
    pm = get_plugin_manager()
    hook = pm.hook
    hook.document_percolated(resource=resource, percolated_queries=percolated_queries)
```

then you can call it here:

```python
if len(percolate_ids) > 0:
    document_percolated_actions(
        resource, 
        PercolateQuery.objects.filter(id__in=percolate_ids)
)